ci-operator/config/openstack-k8s-operators/keystone-operator/openstack-k8s-operators-keystone-operator-master.yaml

```diff
@@ -35,6 +35,18 @@ images:
     RUN pip3 install pre-commit
   from: src
   to: openstack-operator-ci-pre-commit
+- build_args:
+  - name: KUTTL_VERSION
+    value: v1.15.0
+  dockerfile_literal: |
+    FROM openstack-operator-ci-build-deploy
+    ARG KUTTL_VERSION=v0.15.0
+    RUN curl -s -L "https://github.com/kudobuilder/kuttl/releases/download/${KUTTL_VERSION}/kubectl-kuttl_${KUTTL_VERSION#v}_linux_x86_64" -o kubectl-kuttl
+    RUN chmod +x ./kubectl-kuttl
+    RUN mv ./kubectl-kuttl /usr/local/bin
+    RUN yum install -y skopeo
+  from: openstack-operator-ci-build-deploy
```

Comment:
you could continue with `openstack-operator-ci-build-deploy `image only, just adding this additional commands.
But this is up to  you.

Reply:
We can keep this way, it won't change too much from my proposal.